src/main/java/com/google/sps/servlets/OverTimeCasesServlet.java

```diff
@@ -75,6 +75,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Gson gson = new Gson();
       String timeReportJson = gson.toJson(toReturn);
       response.getWriter().println(timeReportJson);
+      // Cases in last 7 days for heatmap
+    } else if (lat == 1000.0 && lng == 1000.0) {
+      List<recentReport> recentReports = new ArrayList<recentReport>();
+      for (LocLatLng gkey : globalTimeReports.keySet()) {
+        if (gkey.location.contains("US")) {
+          continue;
+        }
+        int casesSum = 0;
+        int arrSize = globalTimeReports.get(gkey).size();
+        for (int i = arrSize - 7; i < arrSize; ++i) {
+          casesSum += (globalTimeReports.get(gkey).get(i) - globalTimeReports.get(gkey).get(i - 1));
+        }
+        if (casesSum < 0) {
```

Comment:
Is this situation possible?

Reply:
This can happen if a government removes false positive test results